sdk/test-utils/recorder/src/index.ts

```diff
@@ -6,6 +6,6 @@ export { env, delay, isPlaybackMode, isRecordMode } from "./utils";
 export {
   setReplaceableVariables,
   setReplacements,
-  setEnviromentOnLoad,
+  setEnvironmentOnLoad,
```

Comment:
I'm confused. This reference appears to be correct on master: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/test-utils/recorder/src/index.ts#L9

Reply:
Oh, here's where I needed to update it: https://github.com/Azure/azure-sdk-for-js/pull/5392#pullrequestreview-297564344